### PR TITLE
CSCQVAIN-124: Update info texts

### PR DESCRIPTION
**Pull request by tahme**

### CODE REVIEW

File: src/schemas/fairdata-ida.ui.js [hunk 1]
```diff
@@ -171,8 +171,8 @@ export default {
 		},
 		'placeholder': "– choose location –",
 		'label': "location",
-		'description': "This is some fancy optional description for the location field",
-		'help': "This is the optional help text for the location field",
+		'description': "Location for the dataset (YSO places), e.g. location of observations.",
```
Comment (jppiiroinen) [inline src/schemas/fairdata-ida.ui.js:174]: is this YSO or KOKO?

**Review by jppiiroinen — CHANGES_REQUESTED** [review]
Check if there is a need to replace YSO with KOKO elsewhere too

File: src/schemas/fairdata-ida.ui.js (hunk 1 above)
Comment (tahme) [reply src/schemas/fairdata-ida.ui.js:174]: My understanding is that KOKO includes YSO, but YSO places seems to be a distinct thing, so YSO places is most likely correct.

File: src/schemas/fairdata-ida.ui.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ export default {
 	'/properties/theme': {
 		'tab': 'description',
 		'title': "Subject heading",
-		'description': "Choose subject headings from the General Finnish Ontology (YSO). It also has English and Swedish translations of the terms.",
+		'description': "Choose subject headings from the KOKO Ontology. It also has English and Swedish translations of the terms.",
```
Comment (jppiiroinen) [inline src/schemas/fairdata-ida.ui.js:103]: was this change from YSO to KOKO correct?

Comment (tahme) [reply src/schemas/fairdata-ida.ui.js:103]: Yes.

**Review by jppiiroinen — APPROVED** [review]

**Pull request merged**